src/sonic-config-engine/config_samples.py

```diff
@@ -47,49 +48,105 @@ def generate_empty_config(data):
         new_data['DEVICE_METADATA']['localhost']['type'] = 'LeafRouter'
     return new_data
 
+def port_is_uplink(port_info):
+    return len(port_info['lanes'].split(',')) == 4
+
+def port_is_downlink(port_info):
+    return len(port_info['lanes'].split(',')) == 2
+
+def get_uplinks_downlinks(port_table):
```

Comment:
Please have a comment on the logic here, may be quoting an example as well for clarity.

Reply:
method is removed in the latest update